### PR TITLE
Expose `WhatsNew.shouldPresent`

### PR DESCRIPTION
**Pull request by nixzhu**
I think it's better if we can check shouldPresent before initial a new WhatsNewViewController

### CODE REVIEW

File: Example/WhatsNew/ViewController.swift [hunk 1]
```diff
@@ -14,20 +14,26 @@ class ViewController: UIViewController {
...
+                WhatsNewItem.image(title: "Nice Icons", subtitle: "Completely customize colors, texts and icons.", image: #imageLiteral(resourceName: "love")),
+                WhatsNewItem.image(title: "Such Easy", subtitle: "Setting this up only takes 2 lines of code, impressive you say?", image: #imageLiteral(resourceName: "threed")),
+                WhatsNewItem.image(title: "Very Sleep", subtitle: "It helps you get more sleep by writing less code.", image: #imageLiteral(resourceName: "night")),
+                WhatsNewItem.text(title: "Text Only", subtitle: "No icons? Just go with plain text."),
+                ])
+            whatsNew.titleText = "What's New"
+            whatsNew.itemSubtitleColor = .darkGray
+            whatsNew.buttonText = "Continue"
+            present(whatsNew, animated: true, completion: nil)
+            //whatsNew.presentIfNeeded(on: self)
```
Comment (BalestraPatrick) [inline Example/WhatsNew/ViewController.swift]: Can you please remove this comment too?

**Review by BalestraPatrick — CHANGES_REQUESTED** [review]

**Comment by nixzhu** [comment]
Removed that commented line already.

**Comment by BalestraPatrick** [comment]
Thank you @nixzhu! I am going to release a new version right now 😄

**Comment by nixzhu** [comment]
😊

**Pull request merged**